README.md

```diff
@@ -135,127 +203,68 @@ Comprehensive documentation is available in the [docs/](docs/) directory:
 
-## Commands
-
-### Main commands
-
-| Command | Description |
-|---------|-------------|
-| `dr auth` | Authentication management (login, logout, set-url). |
-| `dr templates` | Template operations (list, clone, setup, status). |
-| `dr run` | Execute application tasks. |
-| `dr dotenv` | Manage environment variables interactively. |
-| `dr self` | CLI utility commands (update, version, completion). |
-| `dr completion` | Generate shell completion scripts. |
-| `dr version` | Show version information. |
+<!--
 
-### Examples
+### Project structure
 
-```bash
-# Authentication
-dr auth login
-dr auth logout
-dr auth set-url https://app.datarobot.com
-
-# Template management
-dr templates list                    # List available templates.
-dr templates clone TEMPLATE_NAME     # Clone a specific template.
-dr templates setup                   # Interactive template setup wizard.
-dr templates status                  # Show current template status.
-
-# Environment configuration
-dr dotenv                           # Interactive environment editor.
-dr dotenv --wizard                  # Configuration wizard mode.
-
-# Task execution
-dr run --list                       # List available tasks.
-dr run dev                          # Run development server.
-dr run build deploy --parallel      # Run multiple tasks in parallel.
-dr run test --watch                 # Run tests in watch mode.
-
-# CLI management
-dr self update                      # Update CLI to latest version.
-dr self version                     # Show version information.
-
-# Get help
-dr --help
-dr templates --help
-dr run --help
 ```
-
-## Shell completion
-
-The CLI supports shell completions for Bash, Zsh, Fish, and PowerShell with automatic installation and configuration.
-
-### Quick setup (Recommended)
-
-The easiest way to install completions is using the interactive installer:
-
-```bash
-# Install completions for your current shell
-dr completion install --yes
-
-# Preview what would be installed (default behavior)
-dr completion install
-
-# Install for a specific shell
-dr completion install bash --yes
-dr completion install zsh --yes
+.
+├── cmd/                    # Command implementations.
+│   ├── auth/              # Authentication commands.
+│   ├── completion/        # Shell completion.
+│   ├── dotenv/            # Environment management.
+│   ├── run/               # Task runner.
+│   ├── templates/         # Template commands.
+│   └── version/           # Version command.
+├── internal/              # Private application code.
+│   ├── config/           # Configuration management.
+│   ├── drapi/            # DataRobot API client.
+│   ├── envbuilder/       # Environment builder.
+│   ├── task/             # Task discovery and execution.
+│   └── version/          # Version information.
+├── tui/                   # Terminal UI components.
+├── docs/                  # Documentation.
+└── main.go               # Application entry point.
```

Comment:
Not sure how important it is but this is out of date:

<img width="335" height="753" alt="Screenshot 2025-11-13 at 4 51 09 PM" src="https://github.com/user-attachments/assets/51bdc05a-7e68-4079-b7c4-2679b0af0cc5" />